### PR TITLE
added roadmap information

### PR DESCRIPTION
**Pull request by LongLiveCHIEF**
Included some bits about our roadmaps and projects, and added a bit more about the intent of our fork moving forward.

One distinction I think it is important to make from the original fork, is that instead of this being a" HTML based" presentation framework, it's a "web-based" presentation framework. I think as we modernize this library, more features like the existing markdown support will allow this library to use many markup formats
and templating tools to create presentation slides and content.

I also updated the License copyright line that we missed with the LICENSE file update, and added some EOL returns for readability in non-wrapping editors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,20 @@
 # reveal.js 
 
-A framework for easily creating beautiful presentations using HTML. [Check out the live demo](http://revealjs.com/).
+A framework for easily creating beautiful web-based presentations. [Check out the live demo](http://revealjs.com/).
 
->__Note:__ This repository was forked from [hakimel/reveal.js](https://github.com/hakimel/reveal.js) on 3/11/2019 to continue active development and feature contributions to reveal.js. Upstream This fork [is published](https://npmjs.com/package/@h3/reveal.js) on npmjs under the `@h3` scope. See [CONTRIBUTING](CONTRIBUTING.md) for info on how to join [HackerHappyHour](https://github.com/HackerHappyHour) and contribute!
+>__Note:__ This repository was forked from [hakimel/reveal.js](https://github.com/hakimel/reveal.js) on 3/11/2019 
+to continue active development and feature contributions to reveal.js. Upstream of this fork
```
Comment (nickshine) [inline README.md]: Not sure how I got the 'Upstream' word mixed in there.. it doesn't make sense when you read the full sentence.
```suggestion
to continue active development and feature contributions to reveal.js. This fork 
```

**Review by cujarrett — APPROVED** [review]
👍

**Comment by cujarrett** [comment]
> looks to pull as many existing PR's from the upstream project as possible, and bring the reveal project up to speed with modern tooling, api design, and front-end features.

This is great!

File: README.md [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
+See [CONTRIBUTING](CONTRIBUTING.md) for info on how to join [HackerHappyHour](https://github.com/HackerHappyHour) and contribute!
 
 ## @h3/reveal.js Roadmap:
-* Native Docker integration
-* ... 
+
+The goal of @h3/reveal moving forward is to make it easier to use for non-developers and
+developers alike by modernizing how reveal is distributed, deployed, extended, and integrated.
+
+Our most important goals are currently tracked in our [projects][] and [milestones][]. In particular the [v2 milestone][]
```
Comment (nickshine) [inline README.md:15]: Either need `[v2 roadmap]` ref link here or change the link definition at bottom to `[v2 milestone]`.
```suggestion
Our most important goals are currently tracked in our [projects][] and [milestones][]. In particular the [v2 milestone][v2 roadmap]
```

File: README.md [hunk 3]
```diff
@@ -1321,3 +1329,8 @@ If you want to include math inside of a presentation written in Markdown you nee
 MIT licensed
 
 Copyright (C) 2018 Hakim El Hattab, http://hakim.se
```
Comment (nickshine) [inline README.md]: Adding 2 spaces for proper newline before added "Modified work  ..." line.
```suggestion
Copyright (C) 2018 Hakim El Hattab, http://hakim.se   
```

**Review by nickshine — CHANGES_REQUESTED** [review]
@LongLiveCHIEF just a few minor things I caught looking at the file in preview mode.

**Review by nickshine — APPROVED** [review]

**Pull request merged**